chain/ethereum/src/chain.rs

```diff
@@ -409,17 +437,35 @@ impl Blockchain for Chain {
         deployment: DeploymentLocator,
         store: impl DeploymentCursorTracker,
         start_blocks: Vec<BlockNumber>,
-        filter: Arc<Self::TriggerFilter>,
+        source_subgraph_stores: Vec<(DeploymentHash, Arc<dyn WritableStore>)>,
```

Comment:
Do we need a writable store to be used here? Is there another trait we could use with less access?

Reply:
Addressed in latest commits